packages/@aws-cdk/cdk/lib/cloudformation/resource.ts

```diff
@@ -229,6 +232,29 @@ export class Resource extends Referenceable {
   }
 }
 
+export class TaggableResource extends Resource implements ITaggable {
+  /**
+   * TagManager to manage the propagation and assignment of tags
+   */
+  public readonly tags: TagManager;
+
+  constructor(parent: Construct, name: string, props: ResourceProps) {
```

Comment:
Shouldn't `props` have `tags` in them for initial tags?

Reply:
I'm assuming we discuss this in detail below.